ghost/core/test/unit/api/canary/utils/serializers/input/pages.test.js

```diff
@@ -30,7 +30,7 @@ describe('Unit: endpoints/utils/serializers/input/pages', function () {
             frame.options.filter.should.eql('(status:published+tag:eins)+type:page');
         });
 
-        it('combine filters', function () {
+        it('only tag filters', function () {
```

Comment:
[To help reviewers] The [page filter](https://github.com/TryGhost/Ghost/pull/14829/files) was removed, so changed the description accordingly;